> Failure in the non-db tests

Running this test in main, and it failed with the same error:

`FAILED tests/core/test_otel_logger.py::TestOtelMetrics::test_timer_start_and_stop_manually_send_true - assert 3140.0 == 3.14`

Maybe @ferruzzi can help us figure it out?

--- a/airflow/config_templates/config.yml
+++ b/airflow/config_templates/config.yml
@@ -1254,7 +1254,7 @@ metrics:
     otel_service:
       description: |
         The default service name of traces.
-      version_added: 2.10.1
+      version_added: 2.10.3
       type: string
       example: ~
       default: "Airflow"

--- a/airflow/config_templates/config.yml
+++ b/airflow/config_templates/config.yml
@@ -1251,6 +1251,13 @@ metrics:
       type: string
       example: ~
       default: "False"
+    otel_service:
+      description: |
+        The default service name of traces.
+      version_added: 2.10.1
+      type: string
+      example: ~
+      default: "Airflow"
     otel_ssl_active:
       description: |
         If ``True``, SSL will be enabled. Defaults to ``False``.

--- a/airflow/metrics/otel_logger.py
+++ b/airflow/metrics/otel_logger.py
@@ -408,8 +408,9 @@ def get_otel_logger(cls) -> SafeOtelLogger:
     # PeriodicExportingMetricReader will default to an interval of 60000 millis.
     interval = conf.getint("metrics", "otel_interval_milliseconds", fallback=None)  # ex: 30000
     debug = conf.getboolean("metrics", "otel_debugging_on")
+    service_name = conf.get("metrics", "otel_service")
 
-    resource = Resource(attributes={SERVICE_NAME: "Airflow"})
+    resource = Resource(attributes={SERVICE_NAME: service_name})
 
     protocol = "https" if ssl_active else "http"
     endpoint = f"{protocol}://{host}:{port}/v1/metrics"

